gslib/commands/rsync.py

```diff
@@ -1052,6 +1055,8 @@ def _CompareObjects(self, src_url_str, src_size, src_mtime, src_crc32c,
     use_hashes = (self.compute_file_checksums or
                   (StorageUrlFromString(src_url_str).IsCloudUrl() and
                    StorageUrlFromString(dst_url_str).IsCloudUrl()))
+    if self.skip_old_files and has_src_mtime and has_dst_mtime and src_mtime < dst_mtime:
```

Comment:
This is predicate is longer than 80 characters; please wrap it in parentheses:

        if (self.skip_old_files and has_src_mtime and has_dst_mtime and
            src_mtime < dst_mtime):
          return False, has_src_mtime, has_dst_mtime